system/Common.php

```diff
@@ -384,22 +373,13 @@ function env(string $key, $default = null)
             return $default;
         }
 
-        // Handle any boolean values
```

Comment:
comment should be copied, this will be covered at Rector side:

- https://github.com/rectorphp/rector-src/pull/5072